.npmrc

```diff
@@ -1 +1,2 @@
+@zimbra:registry=https://npm.pkg.github.com/zimbra
```

Comment:
Would like to understand, why do we want to publish packages at two places? npm and github